tools/testbuild.sh

```diff
@@ -283,25 +301,39 @@ function distclean {
         if [ -d $nuttx/.git ] || [ -d $APPSDIR/.git ]; then
           if [[ -n $(git -C $nuttx status --ignored -s) ]]; then
             git -C $nuttx status --ignored
+            report_test_failure "OS repo not clean."
             fail=1
           fi
           if [[ -n $(git -C $APPSDIR status --ignored -s) ]]; then
             git -C $APPSDIR status --ignored
+            report_test_failure "Apps repo not clean."
             fail=1
           fi
         fi
       fi
     fi
   fi
 
+  if [ ${fail} -eq 0 ]; then
+    report_test_pass
+    # Restore fail flag.
+    fail=$tmp_fail
+  fi
+  end_test_case
+
   return $fail
 }
 
 # Configure for the next build
 
 function configure {
+  start_test_case configure
+  local tmp_fail=$fail
+  fail=0
+
   echo "  Configuring..."
   if ! ./tools/configure.sh ${HOPTION} $config ${JOPTION} 1>/dev/null; then
+    report_test_failure "Failed to configure."
```

Comment:
Please include the config name/option: $config ${JOPTION} in the failure message

Reply:
This is already part of the metadata for the job. I will provide an example once I have the code to render the results. If you want to see an early preview you can download the xml report from the report.  

Also if we really want to have detailed test visibility we should convert the CI running script to use pytest... But my energy for having that argument is really really low.